site/cypress/e2e/Cypress-Feature/superuser_email.spec.js

```diff
@@ -0,0 +1,23 @@
+describe('Superuser Email All Functionality via Sidebar', () => {
+    it('sends an email via Email All and verifies Email Status page', () => {
+        cy.login('superuser');
+
+        cy.get('[data-testid="sidebar"]')
+            .contains('Email All')
+            .click();
+
+        cy.url().should('include', '/superuser/email');
+        cy.get('h1').should('contain', 'System Wide Email');
+
+        const uniqueSubject = `Test Email - ${Date.now()}`;
+        cy.get('#email-subject').type(uniqueSubject);
+        cy.get('#email-content').type('This is a test email sent via Cypress.');
+        cy.get('#send-email').click();
+
+        cy.get('[data-testid="sidebar"]')
+            .contains('Email Status')
+            .click();
+        cy.get('body');
```

Comment:
This line, and the next one don't do anything at the current moment. Are you trying to see if the body contains the subject? If so they should be chained together like `cy.get('body').should('contain', uniqueSubject);`